io.github.yairm210.unciv.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=UnCiv
+Comment=Turn-based historical strategy game
+Comment[fr]=Jeu de stratégie au tour par tour historique
+Icon=io.github.yairm210.unciv
+Exec=unciv
```

Comment:
You probably want to open the unicv-wrapper here?

Reply:
Well, given that I added

```
<provides>
    <binary>unciv</binary>
  </provides>
```

When migrating to metainfo, I wondered if some platforms couldn't parse that to add `unciv-wrapper` to the `$PATH`. With that in mind, I [renamed](https://github.com/flathub/flathub/pull/1339/commits/aeba0564b1b54b9245cf6449cef5b02a250176ba#diff-d83a61e6d8895f78bd2caaea3215d036R29) unciv-wrapper to unciv.

This answer explains why I made those choices. If you feel that your comments still hold and I should:
* remove the `provide` section
* switch back to `unciv-wrapper`

Please tell me, and I will do so.

Reply:
The provide section should be removed as I mentioned before. Never mind this comment, forgot that flatpak modifies the Exec for the exported desktop files.